exampleCourse/questions/demoAutograderNumpy/tests/test.py

```diff
@@ -0,0 +1,12 @@
+from pl_helpers import name, points, not_repeated
+from pl_unit_test import PrairieLearnTestCase
+from code_feedback import Feedback as feedback
```

Comment:
Why are we changing the case of `feedback` on import?

Reply:
The original RELATE grading code used `feedback`.  Since it was a class in the PL implementation, it was capitalized to `Feedback`.  When converting RELATE questions to PL questions, one of the TAs decided it would be better/easier to import this way than to replace `feedback -> Feedback`.

Reply:
Makes sense from a historical perspective, but I think we should use a more standard naming schemes (don't change case on import) in our examples going forwards.